apps/ledger-live-mobile/src/components/RequiresBLE/BluetoothDisabled.tsx

```diff
@@ -11,6 +11,8 @@ const SafeAreaContainer = styled.SafeAreaView`
   align-items: center;
   justify-content: center;
   background-color: ${p => p.theme.colors.background.main};
+  margin-left: 24px;
```

Comment:
Can you the theme spacing values ? 
Ex : `margin-left: ${p => p.theme.space[2]}px;`